### PR TITLE
Added Travis CI

### PR DESCRIPTION
**Pull request by jy300x2**
Added Travis CI to chess app.
Will still need to add rspec and run test.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+before_install: gem update --remote bundler
+install:
+- bundle install --retry=3
+script:
+- bundle exec rspec
+- bundle exec rubocop --config .rubocop.yml
+deploy:
+  provider: heroku
+  run:
+  "rake db:migrate"
```
Comment (Cort98) [inline .travis.yml]: I believe "rake db:migrate" should be right after run:  or indented with a dash like bundle exec rspec.

Comment (mscoutermarsh) [inline .travis.yml]: @jy300x2 ^ yes looks like that is causing the build to fail currently.

Can see the logs here: https://travis-ci.org/Team-Chess-Fusion/chess-fusion/builds/147090274

See if you can get the build to go green. Will probably require some codestyle changes for rubocop to be happy. Let me know if you need help. 👍

**Comment by jy300x2** [comment]
I fixed "rake db:migrate" so that it's indented with a dash. I also added lines to setup the database (postgresql) and lines to run rspec and rubocop.

**Comment by jy300x2** [comment]
Here's what I did for this commit:
- Added rspec
- Deleted test folder
- Added spec files for controllers, models, and helpers
- Wrote a single passing test
- Modified the rails_helper.rb file to have our migrations run automatically

**Comment by mscoutermarsh** [comment]
@jy300x2 looks good! This branch has merge conflicts that you'll need to resolve before you're able to merge.

**Pull request merged**